README.md

```diff
@@ -1,4 +1,3 @@
-![Cloudogu logo](https://cloudogu.com/images/logo.png)
```

Comment:
Please also add the new block for the readme at the end of the file. This is important as it contains the legal notice.

Reply:
done